packages/foreman/katello-certs-tools/katello-certs-tools.spec

```diff
@@ -2,40 +2,63 @@ Name:      katello-certs-tools
 Summary:   Katello SSL Key/Cert Tool
 Group:     Applications/Internet
 License:   GPLv2
-Version:   2.4.0
+Version:   2.5.0
 Release:   1%{?dist}
 URL:       https://github.com/katello/katello-certs-tools
-Source0:   https://codeload.github.com/Katello/%{name}/tar.gz/%{version}
+Source0:   https://codeload.github.com/Katello/%{name}/tar.gz/%{version}#/%{name}-%{version}.tar.gz
 BuildArch: noarch
 
-#BZ 1022017
-Requires: openssl >= 1.0.1e-16
-
+Requires: openssl
 Requires: rpm-build
 
 BuildRequires: docbook-utils
-BuildRequires: python
+BuildRequires: /usr/bin/pathfix.py
+%if 0%{?rhel} > 7
+BuildRequires: python3-devel
+%else
+BuildRequires: python-devel >= 2.6
+%endif
 
 %description
 This package contains tools to generate the SSL certificates required by
 Katello.
 
 %prep
 %setup -q
+%if 0%{?rhel} > 7
+pathfix.py -pni "%{__python3} %{py3_shbang_opts}" . katello-ssl-tool
+%else
+pathfix.py -pni "%{__python2} %{py2_shbang_opts}" . katello-ssl-tool
+%endif
 
 %build
 /usr/bin/docbook2man katello-ssl-tool.sgml
+%if 0%{?rhel} > 7
+%{__python3} setup.py build
+%else
 %{__python} setup.py build
+%endif
 
 %install
+%if 0%{?rhel} > 7
+%{__python3} setup.py install --skip-build --root $RPM_BUILD_ROOT
+chmod +x $RPM_BUILD_ROOT/%{python3_sitelib}/katello_certs_tools/katello_ssl_tool.py
+%else
 %{__python} setup.py install --skip-build --root $RPM_BUILD_ROOT
 chmod +x $RPM_BUILD_ROOT/%{python_sitelib}/katello_certs_tools/katello_ssl_tool.py
+%endif
 mkdir -p $RPM_BUILD_ROOT/%{_sysconfdir}/pki/%{name}
 mkdir -p $RPM_BUILD_ROOT/%{_sysconfdir}/pki/%{name}/certs
 mkdir -p $RPM_BUILD_ROOT/%{_sysconfdir}/pki/%{name}/private
 
 %files
-%{python_sitelib}/*
+%if 0%{?rhel} > 7
+%{python3_sitelib}/katello_certs_tools/*
+%{python3_sitelib}/*.egg-info
```

Comment:
Does py3 still have eggs?

Reply:
No issues with this line

Reply:
I looked it up and you're right. The packaging guidelines suggest being more specific and use `katello_certs_tools-*.egg-info` but this can be good enough.